src/data/finset.lean

```diff
@@ -2973,6 +3007,10 @@ namespace finset
   (⋃ x ∈ (↑s : set α), t x) = ⋃ x ∈ s, t x :=
 rfl
 
+@[simp] theorem bInter_coe (s : finset α) (t : α → set β) :
+  (⋂ x ∈ (↑s : set α), t x) = ⋂ x ∈ s, t x :=
+rfl
+
```

Comment:
Maybe this is out of scope for this PR, but this is being added to a section titled `bUnion`, and most of the lemmas in this section should have corresponding `bInter` statements, not just this one.

How about we either add this in this PR, or raise an issue to make this uniform?

Reply:
I added all variants